include/openssl/evp.h

```diff
@@ -372,6 +376,10 @@ int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
 # define         EVP_CTRL_GET_IV                         0x26
 /* Tell the cipher it's doing a speed test (SIV disallows multiple ops) */
 # define         EVP_CTRL_SET_SPEED                      0x27
+/* Get the unprotectedAttrs from cipher ctx */
+# define         EVP_CTRL_PROCESS_UNPROTECTED            0x28
```

Comment:
Why is this different from something like signerinfo attributes which has an API to handle signed and unsigned attributes?

Reply:
In the case of signerinfo we have a set of attributes that may be added or not added deliberately. In the case of the GOST ciphers, these attributes are properties of ciphers.